src/exception/ExceptionMessages.kt

```diff
@@ -0,0 +1,10 @@
+package exception
+
+class ExceptionMessages {
+    companion object {
+        const val WALL_FEEDBACK = "You cannot go forward, as you have hit a wall."
```

Comment:
these are better as exception subtypes, rather than strings.

Reply:
testing class inheritance is more robust rather than string comparison to programmatically figure out the exact reason of the failure, if any agent wants to.

Reply:
ok